flow_action_components/ChatSessionManagerLA/ChatSessionManagerLAHelper.js

```diff
@@ -0,0 +1,80 @@
+({
+    sendMessage: function(component, event) {
+        var conversationKit = component.find("conversationKit");
+        var sessionId = component.get("v.sessionId");
+        
+        if (sessionId != null && sessionId != '') {
```

Comment:
There's a lot of repeated code in these functions. It would be nice to DRY it up...

Reply:
I have changed the code and added the repeated code lines in the controller instead of helper